src/livekit/MediaDevicesContext.tsx

```diff
@@ -102,27 +111,39 @@ function useMediaDevice(
             // Create a virtual default audio output for browsers that don't have one.
             // Its device ID must be the empty string because that's what setSinkId
             // recognizes.
+            // we also create this if we do not have any available devices, so that
+            // we can use the default or the earpiece.
```

Comment:
I'd be interested to know what platforms/conditions can cause there to be no available devices, because it was useful to have this `available.size` branch to hide the device list in settings during the loading state (while we're still awaiting the `enumerateDevices` call).

Reply:
Thanks for the explanation.
The new change then seems to be the correct solution to me.

With the shortcoming, that we can already see default + earpice while we are still enumerating devices.
But it will remove the "fake default" once we are done enumerating.

To answer your question: I only tested android and it lists not output on enumerate devices.

Reply:
Nice, thanks for adding `available.size` back